### ISSUE TITLE
Solve_Captchas.py > Error : how to debug it ?

### ISSUE THREAD

**Issue by Ricola3D** [MonkeyTedBaker/Monocle#66]
Hello,

I'm trying to solve a captcha from a not banned account (I tried the login with the Pokemon Go app and it worked) but got this error : 
$ python solve_captchas.py
Hash server set to Bossland mode. Please ensure you are using a Bossland NOT a Go Hash hash key.
aiopogo error on <account name>: Unable to parse JSON from hash server.

Was there any recent change in the Hash API ? Or do you know a way to debug it ?

**Comment by MonkeyTedBaker** [comment]
will have a look

**Comment by beloware** [comment]
Yes. Manual solving is not working yet. Try to do, but same error.

**Comment by Ricola3D** [comment]
I used this script on the Monocle/developer branch a few weeks ago successfully with exported X11 display. Thanks Monkey for the help :)
